src/EnergyPlus/WaterCoils.cc

```diff
@@ -2071,14 +2073,16 @@ void InitWaterCoil(EnergyPlusData &state, int const CoilNum, bool const FirstHVA
     // First set the conditions for the air into the coil model
     AirInletNode = state.dataWaterCoils->WaterCoil(CoilNum).AirInletNodeNum;
     WaterInletNode = state.dataWaterCoils->WaterCoil(CoilNum).WaterInletNodeNum;
-    state.dataWaterCoils->WaterCoil(CoilNum).InletAirMassFlowRate = Node(AirInletNode).MassFlowRate;
-    state.dataWaterCoils->WaterCoil(CoilNum).InletAirTemp = Node(AirInletNode).Temp;
-    state.dataWaterCoils->WaterCoil(CoilNum).InletAirHumRat = Node(AirInletNode).HumRat;
-    state.dataWaterCoils->WaterCoil(CoilNum).InletAirEnthalpy = Node(AirInletNode).Enthalpy;
+    auto const &airInletNode = state.dataLoopNodes->Node(AirInletNode);
+    state.dataWaterCoils->WaterCoil(CoilNum).InletAirMassFlowRate = airInletNode.MassFlowRate;
```

Comment:
add a shortcut for `auto &waterCoil` also.

Reply:
Definitely. Shortcuts and removal of using statement are planned for the second pass.